packages/fx-core/resource/package.nls.json

```diff
@@ -111,6 +111,17 @@
   "plugins.spfx.import.success": "Your SPFx solution has been successfully imported to %s.",
   "plugins.spfx.import.log.success": "Teams Toolkit has imported your SPFx solution successfully. A complete log of import details can be found in %s.",
   "plugins.spfx.import.log.fail": "Teams Toolkit failed to import your SPFx solution. A complete log of import details can be found in %s.",
+  "plugins.spfx.addWebPart.confirmInstall": "Teams Toolkit detected your SPFx solution is %s, but you don't have this version installed on your machine. Would you like to install SPFx %s in Teams Toolkit directory to continue adding web part?",
+  "plugins.spfx.addWebPart.install": "Install",
+  "plugins.spfx.addWebPart.confirmUpgrade": "Teams Toolkit is using SPFx %s, but detected your SPFx solution is %s. Would you like to upgrade SPFx version in Teams Toolkit directory to %s and add web part?",
+  "plugins.spfx.addWebPart.upgrade": "Upgrade",
+  "plugins.spfx.addWebPart.versionMismatch.continueConfirm": "Teams Toolkit detected your SPFx solution is %s, but we cannot find this version in your machine. Teams Toolkit uses SPFx installed in Teams Toolkit directory by default (%s). The version mismatch may cause unexpected error. Would you like to continue?",
```

Comment:
Teams Toolkit detected your SPFx solution is %s, but we cannot find this version in your machine. -> SPFx version in your solution is %s, not yet installed on this machine.

Would you like -> Do you want